msalautomationapp/src/main/java/com/microsoft/identity/client/msal/automationapp/sdk/MsalSdk.java

```diff
@@ -233,4 +248,21 @@ public void onError(MsalException exception) {
             throw new AssertionError(exception);
         }
     }
+
+    private IAccount getAccountForPolicyName(@NonNull final MultipleAccountPublicClientApplication pca, @NonNull final String policyName)
+    {
+        try {
+            List<IAccount> accounts = pca.getAccounts();
+            for(IAccount account : accounts)
+            {
+                if (account.getClaims().get("tfp").equals(policyName))
```

Comment:
This is to avoid potential NPE

```suggestion
                if (policyName.equals(account.getClaims().get("tfp")))
```

Reply:
Solve here
c4f4affbab7bbf45d59c1f18fbf89de7db4f2add

Reply:
c4f4affbab7bbf45d59c1f18fbf89de7db4f2add